linode/manager
# refactor: [M3-6400, M3-6401, M3-6404, M3-6408] - MUI v5 Migration - Components > Table, TableBody, TableHead, TableCell, TableSortCell, TableRow

## Description 📝

- MUI v5 Migration for most Table related components 

## Major Changes 🔄
- Runs the TSS migration on Table, TableCell, TableSortCell, TableRow
- Removes `src/components/core/[Table].ts` files because they are a duplicate point of abstraction and a potential way for developers to import an unstyled/un-linode-ifyed component

## Preview 📷

![Screenshot 2023-05-03 at 4 56 39 PM](https://user-images.githubusercontent.com/115251059/236047914-b6566882-022e-4d3a-a2d9-df296cd3feb9.jpg)

## How to test 🧪
- Check different tables throughout the app for styling issues
- I recommend opening production in one tab and this PR in another and comparing the table styles 
- Verify CI ad e2es pass
